lib/Doctrine/Migrations/SchemaDumper.php

```diff
@@ -95,9 +113,23 @@ public function dump(
         $down = implode("\n", $down);
 
         return $this->migrationGenerator->generateMigration(
-            $versionNumber,
+            $fqcn,
             $up,
             $down
         );
     }
+
+    /**
+     * @param string[] $excludedTablesRegexes
+     */
+    private function shouldSkipTable(Table $table, array $excludedTablesRegexes) : bool
+    {
+        foreach (array_merge($excludedTablesRegexes, $this->excludedTablesRegexes) as $regex) {
+            if (preg_match($regex, $table->getName()) !== 0) {
```

Comment:
Should there be preg error handling here? Or are the regexes already validated at this point?

Reply:
No, regex comes out of a console command line parameter. Which validation should be performed? how?

Reply:
"validated" is probably a poor choice of words on my part. I was thinking about something like this: https://github.com/symfony/symfony/blob/62216ea67762b18982ca3db73c391b0748a49d49/src/Symfony/Component/Yaml/Parser.php#L1072-L1090

Reply:
Done in https://github.com/doctrine/migrations/pull/858/commits/dc174a439bc01baa7571bdcdc173e935b4fbf280 . Are you ok with it?